tests/cases/fourslash/getOccurrencesYield.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='fourslash.ts' />
+
+////function* f() {
+//// [|yield|] 100;
+//// [|y/**/ield|] [|yield|] 200;
+////  class Foo {
+////      *memberFunction() {
+////          return yield 1;
+////      }
+////  }
+////  return function* g() {
+////    yield 1;
+////  }
+////}
+
+
+verify.rangesAreOccurrences(false);
+
+goTo.marker();

```

Comment:
Don't think the rest of the test is necessary -- `verify.rangesAreOccurrences` should already loop over all ranges and test that all the other ranges are document highlights of it. See `verifyRangesAreOccurrences` in `fourslash.ts`.

Reply:
I saw this pattern in many fourslash tests so I just copy & paste it. Maybe we should remove these redundant/misleading code.

Reply:
:+1: go for it